rpmbuild/spdk.spec

```diff
@@ -8,6 +8,9 @@
 %{!?requirements:%define requirements 0}
 %{!?build_requirements:%define build_requirements 0}
 %{!?shared:%define shared 0}
+%{!?rbd:%define rbd 0}
+%{!?libdir:%define libdir /usr/local/lib}
+%{!?vfio_user:%define vfio_user 0}
```

Comment:
Same here.

Reply:
Will test the spdk build after removing these lines 
Working on setting up a ceph cluster where I can test the gateway